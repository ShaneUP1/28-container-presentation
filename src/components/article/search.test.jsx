import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SearchBox from './search';

describe('search component', () => {
  afterEach(() => cleanup());
  it('submits search terms', () => {
    const { asFragment } = render(<SearchBox
      onSubmit={jest.fn()} handleChange={jest.fn()}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
