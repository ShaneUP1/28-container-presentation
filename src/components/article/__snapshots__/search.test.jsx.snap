// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search component submits search terms 1`] = `
<DocumentFragment>
  <form>
    <input
      placeholder="What do you want to know?"
      type="text"
    />
    <input
      type="submit"
      value="Search"
    />
  </form>
</DocumentFragment>
`;
